shared/components/SpeakerListIntro/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const SpeakersListIntro = () => {
+  return (
+    <div>
+      <h1>On Stage</h1>
+      <p>
+        From the community. From the community.
+        From the community. From the community.
+        From the community. From the community.
+        From the community. From the community.
+        From the community.
+      </p>
+    </div>
+  );
+};
```

Comment:
That content might become editable? I'm good with either way, just felt like a good separation point.